multichain-testing/scripts/install.sh

```diff
@@ -84,44 +86,42 @@ function install_chart() {
 
 while [ $# -gt 0 ]; do
   case "$1" in
-    -c|--config)
-      CONFIGFILE="$2"
-      shift 2 # past argument=value
-      ;;
-    -v|--version)
-      HELM_CHART_VERSION="$2"
-      shift 2 # past argument
-      ;;
-    -t|--timeout)
-      TIMEOUT="$2"
-      shift 2 # past argument
-      ;;
-    -n|--name)
-      HELM_NAME="$2"
-      shift 2 # past argument
-      ;;
-    --namespace)
-      NAMESPACE="$2"
-      shift 2 # past argument
-      ;;
-    --chart)
-      HELM_CHART="$2"
-      shift 2 # past argument
-      ;;
-    --dry-run)
-      DRY_RUN=1
-      shift # past argument
-      ;;
-    -*|--*)
-      echo "Unknown option $1"
-      exit 1
-      ;;
-    *)
-      ;;
+  -c | --config)
```

Comment:
is this formatting the result of https://github.com/Agoric/agoric-sdk/blob/master/.prettierrc.json5#L23-L29 ?

Reply:
Not sure, doesn't happen for me, but I fixed up the diff

Reply:
Only `config.yaml` changes to single quotes if we do:
```diff
diff --git a/package.json b/package.json
index 40bce1032..39be23458 100644
--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
     "create-agoric-cli": "node ./scripts/create-agoric-cli.cjs",
-    "format": "yarn prettier --write .github golang packages scripts",
+    "format": "yarn prettier --write .github golang packages scripts multichain-testing",
     "lint:format": "yarn prettier --check .github golang packages scripts",
```

This PR is already in the merge queue, but I think someone can pick up this change the next time they're contributing to this directory